server/admin_management/api/app/schemas.py

```diff
@@ -96,3 +152,41 @@ class FamAppAdminGet(BaseModel):
     application: FamApplicationBase
 
     model_config = ConfigDict(from_attributes=True)
+
+
+# -------------------------- FAM Access Control Privilege (Delegated Admin) -------------------------- #
+class FamAccessControlPrivilegeCreateRequest(BaseModel):
+    user_name: Annotated[str, StringConstraints(min_length=3, max_length=20)]
+    user_type_code: famConstants.UserType
+    role_id: int
+    forest_client_number: Union[
```

Comment:
Hi Catherine, I think I remembered you mentioned before you are doing (this time) a bulk creation now.
In this case, could we have `forest_client_numbers` as this is now a List of `forest_client_number`?